sfepy/discrete/fem/fields_base.py

```diff
@@ -1061,19 +1059,21 @@ def create_mapping(self, region, integral, integration,
                 else:
                     se_bf_bg, se_ebf_bg, se_conn = None, None, None
 
-                sg = mapping.get_mapping(qp.vals[0], qp.weights,
-                                         poly_space=Struct(n_nod=bf.shape[-1]),
-                                         extra=(se_conn, se_bf_bg, se_ebf_bg))
+                n_ep = bf.shape[-1]
+                out = mapping.get_mapping(qp.vals[0], qp.weights,
+                                          poly_space=Struct(n_nod=n_ep),
```

Comment:
I wonder if, instead of the fake poly_space, something better could be used - either a true PolySpace instance or None. Maybe None, because `n_ep == se_conn.shape[1]`?

Reply:
This is solved by the next commit.